### ISSUE TITLE
Customers do not reach the thank you page v1.8.0

### ISSUE THREAD

**Issue by stefanliden** [Billmate/magento-2-billmate-checkout#126]
Customers who use Magento: 2.4.1 & PLUGIN: 1.8.0 has reportet that they have customers who never comes to the thank you page and that orders becomes canceled in Online Billmate.

Merchant error log///

[billmate log](https://github.com/Billmate/magento-2-billmate-checkout/files/5836938/billmate.log)
[system log.log](https://github.com/Billmate/magento-2-billmate-checkout/files/5836939/system.log.log)

**Comment by AlldoGroup** [comment]
@andyjohansson 

Any news on this issue?

**Comment by AlldoGroup** [comment]
@magnusemanuelsson and @andyjohansson 

Any news or updates on this issue?

**Comment by andyjohansson** [comment]
Hi @AlldoGroup we are still working on this. We are however approaching a beta for this. Are you interested in trying it out?

**Comment by AlldoGroup** [comment]
@andyjohansson Yes we can try it, as we havent moved any of the critical web sites yet. Just note that we are on the latest version 2.4.2 now. 👍

**Comment by AlldoGroup** [comment]
Any new news or updates on this issue?
